sql/core/benchmarks/JsonBenchmark-results.txt

```diff
@@ -3,128 +3,129 @@ Benchmark for performance of JSON parsing
 ================================================================================================
 
 Preparing data for benchmarking ...
-OpenJDK 64-Bit Server VM 17.0.10+7-LTS on Linux 6.5.0-1016-azure
-AMD EPYC 7763 64-Core Processor
+OpenJDK 64-Bit Server VM 17.0.9+0 on Mac OS X 12.6.7
```

Comment:
Please use GitHub Action's machine to generate this file, and also update the result file for Java 21

Reply:
done.